src/main/java/approximations/java/lang/StringBuffer_Tests.java

```diff
@@ -0,0 +1,2150 @@
+package approximations.java.lang;
+
+
+import approximations.java.Test;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public final class StringBuffer_Tests {
+
+    private static final int MAX_CODE_POINT = 1114111;
+    private static final int MIN_CODE_POINT = 0;
+
+    // internal variables
+
+    // constructors
+
+    // StringBufferAutomaton::StringBuffer
+    @Test(executionMax = 0)
+    public static int test_StringBuffer_0(final int execution) {
+        if (execution == 0) {
+            StringBuffer stringBuffer = new StringBuffer();
+            if (stringBuffer.length() == 0)
+                return 0;
+            else
+                return -1;
+        }
+        return -1;
+    }
+
+
+    // StringBufferAutomaton::StringBuffer (CharSequence)
+    @Test(executionMax = 1)
+    public static int test_StringBuffer_1(final int execution) {
+
+        CharSequence sequence = "12";
+
+        if (execution == 0) {
+            StringBuffer stringBuffer = new StringBuffer(sequence);
+            if (stringBuffer != null && stringBuffer.length() == 2 && stringBuffer.charAt(0) == '1' && stringBuffer.charAt(1) == '2')
+                return 0;
+            else
+                return -1;
+        }
+
+        if (execution == 1) {
+            sequence = null;
+            try {
+                StringBuffer stringBuffer = new StringBuffer(sequence);
+            } catch (NullPointerException e) {
+                return 1;
+            }
+            return -1;
+        }
+        return -1;
+    }
+
+
+    // StringBufferAutomaton::StringBuffer (String)
+    @Test(executionMax = 1)
+    public static int test_StringBuffer_2(final int execution) {
+        String sequence = "12";
+
+        if (execution == 0) {
+            StringBuffer stringBuffer = new StringBuffer(sequence);
+            if (stringBuffer != null && stringBuffer.length() == 2 && stringBuffer.charAt(0) == '1' && stringBuffer.charAt(1) == '2')
+                return 0;
+            else
+                return -1;
+        }
+
+        if (execution == 1) {
+            sequence = null;
+            try {
+                StringBuffer stringBuffer = new StringBuffer(sequence);
+            } catch (NullPointerException e) {
+                return 1;
+            }
+            return -1;
+        }
+        return -1;
+    }
+
+
+    // StringBufferAutomaton::StringBuffer (int)
+    @Test(executionMax = 0)
```

Comment:
If `executionMax` is 0 (zero) then annotation is unnecessary. It will be guessed automatically.